doc/design/milestone-0.0.2/query-api.md

```diff
@@ -45,10 +45,6 @@ Example Usage:
 // meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 d := &apps_v1.Deployment{
-    TypeMeta: meta_v1.TypeMeta{
```

Comment:
I think that we probably want to keep this, it is from an older version we probably don't want to re-write this history IMO

Reply:
Sure.Done 👍